pages/about.html

```diff
@@ -4,9 +4,59 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>About Me</title>
 </head>
-<body>
-  
+<body style="background-color:lightcyan;">
+  <nav>
+    <a href="./index.html">Home</a>
+    <a href="./about.html">About</a>
+    <a href="./portfolio.html">Projects</a>
+    <link href="style.css" rel="stylesheet">
+  </nav>
```

Comment:
Nice `nav`! However, please place links to your CSS in the head of your document.